### PR TITLE
Update linguage português br

### PR DESCRIPTION
**Pull request by Lauro48585995**
Adicionado algumas pontuações e palavras em alguns trechos.

Id:3147928 para tag de tradutor no jogo

### CODE REVIEW

**Review by Lauro48585995 — COMMENTED** [review]
String 793 erro grave aí

**Review by Jeiii20 — CHANGES_REQUESTED** [review]
Lines 699 (`join_clan_war`) and 702 (`banned_until`) don't need any punctuation, please fix it.

**Comment by Jeiii20** [comment]
**Another note:** Don't forget to always explain why the previous text was incorrect. Take this example:

Line 793 (`Survive_a_Zombie_Apocalypse`) is incorrect because, according to ChatGPT's research:

**The grammatically correct translation is "Sobreviva a um Apocalipse Zumbi".

In Portuguese, the verb "sobreviver" requires the preposition "a" when followed by a noun. Therefore, "Sobreviva a um Apocalipse Zumbi" is correct, while "Sobreviva um Apocalipse Zumbi" would be ungrammatical because it omits the necessary preposition.

Even though the first one is more literal, the second option is more appropriate for correct usage in Portuguese.**

Please consider this topic for your next changes. Thanks for your report. ^^

**Review by Lauro48585995 — COMMENTED** [review]
Id3147928 para tag

**Comment by Jeiii20** [comment]
@Lauro48585995 

Please stop messing up too much with the changes. Please undo these changes so them can be elligible. You can check what you did and return as they were.

File: res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -696,10 +696,10 @@
     <string name="member_permissions">"Permissões para Membros"</string>
     <string name="edit">"EDITAR"</string>
     <string name="start_clan_war">"Começar uma Guerra de Clãs"</string>
-    <string name="join_clan_war">"Entrar em Guerra de Clãs"</string>
+    <string name="join_clan_war">"Entrar em Guerra de Clãs."</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:699]:
```suggestion
    <string name="join_clan_war">"Entrar em Guerra de Clãs"</string>

File: res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -696,10 +696,10 @@
...
     <string name="none">"Nenhum"</string>
     <string name="INVALID_TOKEN">"Token Inválido!"</string>
-    <string name="banned_until">"Banido até "</string>
+    <string name="banned_until">"Banido até. "</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:702]:
```suggestion
    <string name="banned_until">"Banido até: "</string>

File: res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -833,7 +833,7 @@
     <string name="UNBLOCK">DESBLOQUEAR</string>
     <string name="Emotes">Mostrar Botão de Emotes</string>
     <string name="Support">Suporte</string>
-    <string name="Already_Blocked">Você Já Bloqueou</string>
+    <string name="Already_Blocked">Você Já Bloqueou!</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:836]:
```suggestion
    <string name="Already_Blocked">Você Já Bloqueou</string>

File: res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -949,10 +949,10 @@
     <string name="Nebulous_Rules" translatable="true" tools:ignore="Untranslatable">
 Regras do Nebulous\n
 \n
-1. Não compartilhe ou negocie contas com ninguém. Isso fará com que ambos os jogadores sejam banidos permanentemente.\n
-2. Não peça contas de ninguém. Isso fará com que você seja banido permanentemente.\n
-3. Não tente se passar por um Moderador ou Youtuber. Isso fará com que você seja banido permanentemente.\n
-4. Não trapaceie para chegar aos Top Classificados do jogo em Guerras de Clã/Arena (FreePoints). Isso redefinirá suas estatísticas e resultará em um banimento permanente.\n
+1. Não compartilhe ou negocie contas com ninguém. Isso fará com que ambos os jogadores sejam banidos permanente.\n
+2. Não peça contas de ninguém. Isso fará com que você seja banido permanente.\n
+3. Não tente se passar por um Moderador ou Youtuber. Isso fará com que você seja banido permanente.\n
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:954]:
```suggestion
1. Não compartilhe ou negocie contas com ninguém. Isso fará com que ambos os jogadores sejam banidos permanentemente.\n
2. Não peça contas de ninguém. Isso fará com que você seja banido permanentemente.\n
3. Não tente se passar por um Moderador ou Youtuber. Isso fará com que você seja banido permanentemente.\n

File: res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -1002,7 +1002,7 @@ Se você notar um jogador quebrando as regras do jogo, denuncie a um Moderador.
     <string name="Second_Pet">Permite você a ter dois mascotes ao mesmo tempo!</string>
     <string name="This_item_requires">Este item requer</string>
     <string name="not_enough_plasma">Não há plasma suficiente.</string>
-    <string name="MARK_UNREAD">MARCAR COMO NÃO LIDA</string>
+    <string name="MARK_UNREAD">MARCAR COMO NÃO LIDA.</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1005]:
```suggestion
    <string name="MARK_UNREAD">MARCAR COMO NÃO LIDA</string>

File: res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -1155,7 +1155,7 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="Not_in_a_group_">Não está em um grupo.</string>
     <string name="group_chat_create" translatable="true" tools:ignore="Untranslatable">Envie este ID do grupo para os seus amigos, então eles poderão entrar no grupo:</string>
     <string name="Specify_Group_ID" translatable="true" tools:ignore="Untranslatable">Especifique o ID do Grupo</string>
-    <string name="Failed">Falhou</string>
+    <string name="Failed">Falhou!</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1158]:
```suggestion
    <string name="Failed">Falhou</string>

File: res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -1223,7 +1223,7 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="Allow_permission_to_enable_voice_recording_">Conceder a permissão para ativar a gravação de voz.</string>
     <string name="voice_chat_prompt">A classificação do conteúdo do aplicativo pode mudar com o chat por voz ativado. Para um áudio melhor, use um fone de ouvido.</string>
     <string name="Only_leader_can_answer_requests_">Apenas o Líder pode responder às solicitações.</string>
-    <string name="Sent_request_not_invited_">Pedido enviado (não convidado).</string>
+    <string name="Sent_request_not_invited_">Pedido enviado! (não convidado).</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1226]:
```suggestion
    <string name="Sent_request_not_invited_">Pedido enviado (você não foi convidado).</string>

File: res/values-pt-rBR/strings.xml [hunk 8]
```diff
@@ -1258,7 +1258,7 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="Collected_">Coletado:</string>
     <string name="Collisions_">Colisões:</string>
     <string name="Absorbed_">Absorvido:</string>
-    <string name="team_warning">A parceria com um jogador que não está na sua equipe resultará em um banimento! Se você ver outras pessoas se unindo em outras equipes, poderá votar para expulsá-la da sala, no menu do jogador.</string>
+    <string name="team_warning">A parceria com um jogador que não está na sua equipe resultará em um banimento,Se você ver outras pessoas se unindo em outras equipes, poderá votar para expulsá-la da sala, no menu do jogador.</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1261]: Please return from comma to exclamation mark on where you changed.

File: res/values-pt-rBR/strings.xml [hunk 9]
```diff
@@ -1307,7 +1307,7 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="Plasma_Boost_2_Hours_">Impulso de Plasma 2 Horas.</string>
     <string name="Plasma_Boost">Impulso de Plasma</string>
     <string name="skin_warning" translatable="true" tools:ignore="Untranslatable">
-        "A sua skin não foi aprovada. Aqui estão algumas das razões mais comuns. Você NÃO deve enviar:
+        "A sua skin não foi aprovada! Aqui estão algumas das razões mais comuns. Você NÃO deve enviar:
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1310]: Please return from exclamation mark to dot (period).

File: res/values-pt-rBR/strings.xml [hunk 10]
```diff
@@ -1366,11 +1366,11 @@ Comandos:
     <string name="MEGA_SPLIT">MEGA SPLIT 64x</string>
     <string name="Missing_Permissions">Permissões ausentes</string>
     <string name="DELETE_ACCOUNT">EXCLUIR CONTA</string>
-    <string name="delete_prompt">Você tem certeza que quer excluir sua conta? Esta ação é PERMANENTE e NÃO PODERÁ SER DESFEITA!</string>
+    <string name="delete_prompt">Você tem certeza que quer excluir sua conta? Esta ação é PERMANENTE e NÃO PODERÁ SER DESFEITA.</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1369]: Please return from period to exclamation mark.

**Review by Jeiii20 — CHANGES_REQUESTED** [review]

File: res/values-pt-rBR/strings.xml [hunk 11]
```diff
@@ -1366,11 +1366,11 @@ Comandos:
...
 
 
     <string name="Type">Digite</string>
-    <string name="Complete_campaign_mission_">Complete a missão de campanha</string>
+    <string name="Complete_campaign_mission_">Complete a missão de campanha.</string>
```
Comment (Jeiii20) [inline res/values-pt-rBR/strings.xml:1373]: Same as line 699

```suggestion
    <string name="Complete_campaign_mission_">Complete a missão de campanha</string>

**Comment by Jeiii20** [comment]
PR closed. Too many inaccuracies regarding punctuations and unresolved conversations.

**Pull request closed without merging**